_includes/master/reqs-sys.md

```diff
@@ -1,9 +1,13 @@
-## Node hosts
+## Node requirements
 
 - AMD64 processor
 
-- Linux kernel 3.10 or later. The following distributions are known to work 
-  well with {{site.prodname}} and {{include.orch}}.
+- Linux kernel 3.10 or later. Kernel dependencies: `nf_conntrack_netlink` subsystem, 
```

Comment:
Should this be a bulleted list or perhaps a table?

Reply:
I feel like this should go lower down on the page - it's _much_ less likely to cause a problem for someone than say, using an unsupported distro or forgetting to allow traffic in firewall rules.